_people/mikhail_titov.md

```diff
@@ -0,0 +1,38 @@
+---
+name: "Mikhail Titov"
+collection: people
+avatar: mikhail_titov.jpg
+joined: 2020
+position: researcher
+title: "Research Associate"
+permalink: /people/mikhail_titov
+contacts:
+  email: mikhail.titov@rutgers.edu
+  office: CoRE 707
+  github: https://github.com/mtitov
+  linkedin: https://linkedin.com/in/mikhail-a-titov
+---
+
+Mikhail is a Research Associate at the Rutgers University. He pursued his Ph.D. 
+degree in Computer Science at the University of Texas at Arlington in 2016, 
+under the supervision of Dr. Gergely Záruba and Dr. Kaushik De. His doctoral 
+research was focused on applying data mining and machine learning techniques 
+and methods to the workload management system PanDA (the Production and 
+Distributed Analysis system) to investigate correlations and similarities 
+between data objects (involved entities, e.g., inter-user, user-data 
+relationships) for efficient data distribution and data caching strategies. 
+In 2007, he received his Combined B.S./M.S. degree in Computer Engineering 
+(Computing and Information Science) at the Moscow Engineering and Physics 
+Institute (NRNU "MEPhI", Moscow, Russia). Since 2008, he is an Associated member 
+of the personnel (Scientist) at the European Organization for Nuclear Research 
+(CERN, Geneva, Switzerland).
```

Comment:
Do you mean "... an associate staff member (scientist)at CERN, the European Organization for Nuclear Research, at Geneva, Switzerland."?

Reply:
that is actually that way how I'm affiliated to CERN (it is "Associated member of the personnel"), staff is who actually employed by CERN